[CUBRIDMAN-145] Restore shard-related content in manual

Kang-dot opened the pull request:
http://jira.cubrid.org/browse/CUBRIDMAN-145

Purpose

- Restore shard-related content in latest menual

Implementation

- N/A

Remarks

- Most of contents is taken from 9.3.0 version manual
- 9.3.0 manual doesn't have english shard content. so I translate the Korean version and used it as an English version.

Kang-dot committed:
[CUBRIDMAN-145] Restore shard-related content in manual

- add shard.rst file
- add shard content in config.rst
- add shard.rst into index's toctree

Kang-dot committed:
[CUBRIDMAN-145] Restore shard-related content in manual

- add shard-relate content in manual
- add english shard menual

kisoo-han commented:
Please check Git history for cubrid-manual for English version.

kisoo-han reviewed (changes requested):
Please change and request review again

Kang-dot commented:
I will request PR again after changing the en/shard.rst file.

Kang-dot committed:
[CUBRIDMAN-145] Restore shard-related content in manual

- change en/shard.rst file. restore from old commit
- fix document's table format

Kang-dot committed:
[CUBRIDMAN-145] Restore shard-related content in manual

- edit en/config.rst
- edit en/control.rst

Kang-dot committed:
[CUBRIDMAN-145] Restore shard-related content in manual

- edit content in en, ko
